### PR TITLE
Pwned v2

### PR DESCRIPTION
**Pull request by ohaim1008**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Enhancement to have I been pawned - v2 with "username" command and converting to pack.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation 

## Notes:
The old Pwned playbook is unused and therefore was delete.

## Technical writer review
Mention and link to the files that require a technical writer review.
- [x] [YAML file](https://github.com/demisto/content/blob/9de4e46964a8467c58e66e35a0c857445d57c221/Packs/PwnedV2/Integrations/PwnedV2/PwnedV2.yml)
- [x] [READ ME](https://github.com/demisto/content/blob/9de4e46964a8467c58e66e35a0c857445d57c221/Packs/PwnedV2/Integrations/PwnedV2/README.md)

fixes: https://github.com/demisto/etc/issues/20226

### CODE REVIEW

File: Packs/PwnedV2/Integrations/PwnedV2/PwnedV2_test.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    - response of the database
+    When
+    - mock the website result
+    Then
+    - convert the result to human readable table
+    - create the context
+    validate the expected_result and the created context
+    """
+    mocker.patch('PwnedV2.http_request', return_value=response)
+    for args in command(args):
```
Comment (bakatzir) [inline Packs/PwnedV2/Integrations/PwnedV2/PwnedV2_test.py]: why is there a for loop? can you explain?

Comment (ohaim1008) [reply Packs/PwnedV2/Integrations/PwnedV2/PwnedV2_test.py]: yield is returning "generator object" in order to get to the item in the object there is a need in loop

File: Packs/PwnedV2/Integrations/PwnedV2/PwnedV2.py [hunk 2]
```diff
@@ -236,36 +243,57 @@ def pwned_email(email, email_suffix, paste_suffix):
...
+        'test-module': test_module,
+        'email': pwned_email_command,
+        'pwned-email': pwned_email_command,
+        'domain': pwned_domain_command,
+        'pwned-domain': pwned_domain_command,
+        'pwned-username': pwned_username_command
+    }
+
+    if command in commands:
+        for args in commands[command](demisto.args()):
```
Comment (bakatzir) [inline Packs/PwnedV2/Integrations/PwnedV2/PwnedV2.py]: why args and not e.g responses?

Comment (ohaim1008) [reply Packs/PwnedV2/Integrations/PwnedV2/PwnedV2.py]: you are right, I'll change it

**Comment by ohaim1008** [comment]
Hi Alex,
Please review the following files:
- YAML file:https://github.com/demisto/content/blob/9de4e46964a8467c58e66e35a0c857445d57c221/Packs/PwnedV2/Integrations/PwnedV2/PwnedV2.yml
- READ ME https://github.com/demisto/content/blob/9de4e46964a8467c58e66e35a0c857445d57c221/Packs/PwnedV2/Integrations/PwnedV2/README.md
Thank you!

File: Packs/PwnedV2/Integrations/PwnedV2/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
--
+Added username command  the option to search by username.
```
Comment (bakatzir) [inline Packs/PwnedV2/Integrations/PwnedV2/CHANGELOG.md]:
```suggestion
Added the **pwned-username*** command the allow searching usernames.
```

File: Packs/PwnedV2/Integrations/PwnedV2/PwnedV2.py [hunk 4]
```diff
@@ -7,6 +7,7 @@
 import re
 import requests
 
+
```
Comment (bakatzir) [inline Packs/PwnedV2/Integrations/PwnedV2/PwnedV2.py]: rm

File: Packs/PwnedV2/Integrations/PwnedV2/PwnedV2.py [hunk 5]
```diff
@@ -217,55 +223,99 @@ def set_retry_end_time():
 ''' COMMANDS + REQUESTS FUNCTIONS '''
 
 
-def test_module():
-    http_request('GET', SAMPLE_TEST_SUFFIX)
-    demisto.results('ok')
+def test_module(args_dict):
+    http_request('GET', SUFFIXES.get("test"))
+    yield ['ok', {}, []]
```
Comment (bakatzir) [inline Packs/PwnedV2/Integrations/PwnedV2/PwnedV2.py]: let's not use the yield here as well.

**Comment by ShirleyDenkberg** [comment]
@ohaim1008 Done. Please regenerate the Readme from the updated yml

**Comment by bakatzir** [comment]
@ShirleyDenkberg Welcome to team! 🎉

**Comment by ShirleyDenkberg** [comment]
Thanks!

Regards,


Shirley Denkberg |  Senior Technical Writer

Palo Alto Networks  |  Yigal Alon 94  |  Tel Aviv  |  Israel

Mobile: +972 584323365  | www.paloaltonetworks.com
<https://www.paloaltonetworks.com/>

File: Packs/PwnedV2/Integrations/PwnedV2/PwnedV2.py [hunk 6]
```diff
@@ -84,13 +83,11 @@ def http_request(method, url_suffix, params=None, data=None):
 
 def html_description_to_human_readable(breach_description):
     """
-
-    Args:
-        breach_description: Description of breach from API response
-
-    Returns: Description string that altered HTML urls to clickable urls
+    Executing the pwned request for usernames list, in order to support list input, the function returns 3 lists of
```
Comment (bakatzir) [inline Packs/PwnedV2/Integrations/PwnedV2/PwnedV2.py:86]: I think you copy-paste here

Comment (ohaim1008) [reply Packs/PwnedV2/Integrations/PwnedV2/PwnedV2.py:86]: by mistake - fixing

**Pull request merged**